openandroidinstaller/views/select_view.py

```diff
@@ -296,11 +301,11 @@ def get_notes(self) -> str:
         notes = []
 
         brand = self.state.config.metadata.get("brand", "")
-        if brand in "xiaomi":
+        if brand == "xiaomi":
             notes.append(
                 "- If something goes wrong, you can reinstall MiUI here:\n<https://xiaomifirmwareupdater.com/>\n"
             )
-        elif brand in "poco":
+        elif brand == "poco":
```

Comment:
Why two conditions? Poco is Xiaomi, `if brand == "xiaomi" or brand == "poco"` works, isn't it?
And please remove `miui/{self.state.config.device_code}/` in the link, you forget this one ^^

Reply:
That looks like an issue introduced by merging with main... I corrected it now